code/__HELPERS/icon_smoothing.dm

```diff
@@ -137,6 +137,10 @@
 	icon_state = ""
 	return TRUE
 
+/obj/effect/corners_diagonal_smooth_effect
+	icon = 'icons/turf/floors.dmi'
+	icon_state = "transparent"
+	vis_flags = VIS_INHERIT_ID|VIS_INHERIT_PLANE|VIS_INHERIT_LAYER|VIS_UNDERLAY
```

Comment:
Could you add this into a path similar to https://github.com/tgstation/TerraGov-Marine-Corps/blob/master/code/game/visual_objects.dm ? Ideally into the `/modules` folder instead of the `/game` one.

Better move this back into movable than into object, to inherit less clutter.

Also, if we could re-use these objects it would reduce memory a lot.

Reply:
I make it re-use objects.